docs/2.guide/1.concepts/3.rendering.md

```diff
@@ -17,48 +17,48 @@ While this technique allows building complex and dynamic UIs with smooth page tr
+- **Search Engine Optimization**: Indexing and updating the content delivered via client-side rendering takes longer than with a server-rendered HTML document. That is related to the performance drawback we discussed, as search engine crawlers won't wait for the interface to be fully rendered on their first try to index the page. Your content will take more time to show and update in search results pages with pure client-side rendering.
 
 ### Examples
 
 Client-side rendering is a good choice for heavily interactive **web applications** that don't need indexing or whose users visit frequently. It can leverage browser caching to skip the download phase on subsequent visits, such as **SaaS, back-office applications, or online games**.
 
 ## Universal Rendering
 
-When the browser requests a URL with universal (client-side + server-side) rendering enabled, the server returns a fully rendered HTML page to the browser. Whether the page has been generated in advance and cached or is rendered on the fly, at some point, Nuxt has run the JavaScript (Vue.js) code in a server environment, producing an HTML document. Users immediately get the content of our application, contrary to client-side rendering. This step is similar to traditional **server-side rendering** performed by PHP or Ruby applications.
+When the browser requests a URL with universal (client-side + server-side) rendering enabled, the server returns a fully rendered HTML page to the browser. Whether the page has been generated in advance and cached or rendered on the fly, at some point, Nuxt has run the JavaScript (Vue.js) code in a server environment, producing an HTML document. Users immediately get the content of our application, contrary to client-side rendering. This step is similar to traditional **server-side rendering** performed by PHP or Ruby applications.
 
-To not lose the benefits of the client-side rendering method, such as dynamic interfaces and pages transitions, the Client loads the javascript code that runs on the Server in the background once the HTML document has been downloaded. The browser interprets it again (hence **Universal rendering**) and Vue.js takes control of the document and enables interactivity.
+To not lose the benefits of the client-side rendering method, such as dynamic interfaces and page transitions, the client loads the JavaScript code that runs on the server in the background once the HTML document has been downloaded. The browser interprets it again (hence **Universal rendering**) and Vue.js takes control of the document and enables interactivity.
 
 Making a static page interactive in the browser is called "Hydration."
```

Comment:
```suggestion
Making a static page interactive in the browser is called hydration.
```

Reply:
It's the name of "phenomenon", why not keep it upper case?